### PR TITLE
Add importlib.resources.files for Python 3.9

### PR DESCRIPTION
**Pull request by hynek**

### CODE REVIEW

**Comment by hynek** [comment]
Do I really have to create an `abc.pyi` and re-export all the abcs/Protocols!?

**Comment by srittau** [comment]
As far as I can see, the problem is simply that `stdlib/3/importlib/abc.pyi` doesn't yet have the `Traversable` protocol.

**Comment by hynek** [comment]
Hmm I got confused by the error that "found module, but no stubs". 🤔

**Comment by hauntsaninja** [comment]
Thanks, this looks good to me! You'll need to add `import abc` to `importlib.abc` (or possibly `from abc import abstractmethod`) and reformat with black

**Comment by hynek** [comment]
I'm somewhat confused by the remaining failures.

**Comment by hynek** [comment]
Thanks @hauntsaninja!

File: stdlib/3/importlib/abc.pyi [hunk 1]
```diff
@@ -70,3 +70,27 @@ if sys.version_info >= (3, 7):
...
+        @abstractmethod
+        def is_dir(self) -> bool: ...
+        @abstractmethod
+        def is_file(self) -> bool: ...
+        @abstractmethod
+        def joinpath(self, child: Traversable) -> Traversable: ...
+        @abstractmethod
+        def __truediv__(self, child: Traversable) -> Traversable: ...
+        @abstractmethod
+        def open(self, mode: str = ..., *args, **kwargs) -> IO: ...
```
Comment (srittau) [inline stdlib/3/importlib/abc.pyi:93]: If I understand the docstring correctly, `mode` would be better annotated as `Literal["r", "rb"]`. Also a nit: `args` and `kwargs` could be annotated as `Any`.

**Review by srittau — COMMENTED** [review]
Thanks, looks mostly good to me with one remark and a nit.

**Comment by hynek** [comment]
lol wat 😅

```
free(): invalid pointer
/home/runner/work/_temp/c2a28e77-7168-488e-8f26-da1cbba538c8.sh: line 1:  2734 Aborted                 (core dumped) ./tests/pytype_test.py
```

I think it needs whitelisting too?

**Review by srittau — APPROVED** [review]

**Comment by srittau** [comment]
:( @hynek Sorry for the hassle. Please just add it to the whilelist for now please. Cc @rchen152: It seems pytype crashes here.

**Comment by rchen152** [comment]
O_o That failure mode I haven't seen before. Sorry for the trouble! I opened https://github.com/google/pytype/issues/757.

**Pull request merged**